src/utils/client.js

```diff
@@ -82,9 +84,54 @@ async function getAccessToken(url) {
       error(`Exception obtaining an access token from ${server.endpoint}\n${e.message}`);
       throw e;
     }
-  } else {
+  } else if (server) {
     return server.token;
+  } else {
+    error(`Unable to get access token for ${url}`);
+    return;
+  }
+}
+
+/**
+ * Register the backend app with the server and add it to the servers collection
+ *
+ * @param {string} url - the server base url
+ * @returns the server object if successful, otherwise null
+ */
+async function registerServer(url) {
+  if (
+    configUtil.getDynamicClientRegistration() &&
+    configUtil.getDynamicClientRegistration() === 'false'
```

Comment:
Wouldn't this work without the `configUtil.getDynamicClientRegistration()`?  If it was null, the equality comparison would still return false.